enhancements/scheduling/clusteroperator-scheduling.md

```diff
@@ -0,0 +1,108 @@
+---
+title: clusteroperator-scheduling
+authors:
+  - "@michaelgugino"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-02-12
+last-updated: 2021-02-12
+status: implementable
+---
+
+# ClusterOperator Scheduling
+
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+To ensure system components, especially ClusterOperators, have the correct
+set of tolerations.
```

Comment:
Possibly PR with adjustments to [this existing enhancements content][1], instead of creating a new, parallel enhancement?

[1]: https://github.com/openshift/enhancements/blob/feed05c5c9838fa9476efd650cc968a15ac6b42b/CONVENTIONS.md#taints-and-tolerations

Reply:
I should have used grep!  I looked in the scheduling folder but didn't find this obviously.